homeassistant/components/rfxtrx.py

```diff
@@ -396,6 +395,12 @@ def __init__(self, name, event, datas, signal_repetitions):
         self._state = datas[ATTR_STATE]
         self._should_fire_event = datas[ATTR_FIREEVENT]
         self._brightness = 0
+        self._added_to_hass = False
+
+    @asyncio.coroutine
+    def async_added_to_hass(self):
+        """Register callbacks."""
+        self._added_to_hass = True
```

Comment:
The device [call back subscription](https://github.com/home-assistant/home-assistant/pull/9117/files#diff-89c8b1b88a53aab1ae938235645d3a59R43) is done only once for all the switches (and the same for lights and sensors), and should not be done for each switch.
So do not no how to solve that?